README.md

```diff
@@ -22,27 +22,34 @@ MLS validation service:
 - 2vCPU
 - 512MiB memory
 
-## Available tooling
+## Get started
 
-- [Helm charts](./helm/README.md) - Deploy xmtpd nodes on Kubernetes clusters
-- [Terraform](./terraform/README.md) - Provision cloud infrastructure for xmtpd nodes
+1. Provision your infrastructure.
 
-## Production deployment guide for GKE
+   You can use this [Terraform tooling](/terraform/) if you need to provision underlying cloud infrastructure.
```

Comment:
not quite right.

The terraform stuff is for AWS/ECS 0-1.
The helm stuff is for already deployed Kubernetes.

They do not build one on top of the other